src/pages/Checkout/CheckoutPage.vue

```diff
@@ -551,19 +543,7 @@ export default {
 	},
 	methods: {
 		loginToContinue(event) {
```

Comment:
Please remove this function entirely, as well as the calls to it on lines 121 and 134

Reply:
Done!